### ISSUE TITLE
modify the parameters

### ISSUE THREAD

**Issue by ppl520zfl** [bowenc0221/panoptic-deeplab#11]
Hi,bowen,Thanks for your great work!
I want to know how to modify the parameters if  I don't use pre-training . Experiments show that if only use the default parameters (just set pretrain=False), PQ is only about 45% on cityscapes.

**Comment by bowenc0221** [comment]
I would suggest you change MAX_ITER from 90k to 900k. If it still cannot get the 59%, then try also increasing the learning rate by 10x.

**Comment by ppl520zfl** [comment]
> I would suggest you change MAX_ITER from 90k to 900k. If it still cannot get the 59%, then try also increasing the learning rate by 10x.

Thanks for your reply!
